platform-api/che-core-api-user/src/main/java/org/eclipse/che/api/user/server/UserService.java

```diff
@@ -312,6 +316,52 @@ public void remove(@ApiParam(value = "User ID")
         userDao.remove(id);
     }
 
+
+    /**
+     * Allow to check if current user has a given role or not. status <b>200</b> and {@link UserInRoleDescriptor} is returned by indicating if role is granted or not
+     *
+     * @param role
+     *         role to search (like admin or manager)
+     * @param scope
+     *         the optional scope like system, workspace, account.(default scope is system)
+     * @param scopeId
+     *         an optional scopeID used by the scope like the workspace ID if scope is workspace.
+     * @return {UserInRoleDescriptor} which indicates if role is granted or not
+     * @throws org.eclipse.che.api.core.ForbiddenException
+     *         with an uknown scope
+     * @throws ServerException
+     *         when unable to perform the check
+     */
+    @ApiOperation(value = "Check role for the authenticated user",
+            notes = "Check if user has a role in given scope (default is system) and with an optional scope id. Roles allowed: user, system/admin, system/manager.",
+            response = UserInRoleDescriptor.class,
+            position = 7)
+    @ApiResponses({@ApiResponse(code = 200, message = "OK"),
+                   @ApiResponse(code = 403, message = "Unable to check for the given scope"),
+                   @ApiResponse(code = 500, message = "Internal Server Error")})
+    @GET
+    @Path("/inRole")
+    @GenerateLink(rel = LINK_REL_GET_USER_BY_ID)
+    @RolesAllowed({"user", "system/admin", "system/manager"})
+    @Produces(APPLICATION_JSON)
+    public UserInRoleDescriptor inRole(@Required @Description("role inside a scope") @QueryParam("role") String role,
+                                       @DefaultValue("system") @Description("scope of the role (like system, workspace)") @QueryParam("scope") String scope,
+                                       @DefaultValue("") @Description("id used by the scope, like workspaceId for workspace scope") @QueryParam("scopeId") String scopeId,
+                                       @Context SecurityContext context) throws NotFoundException, ServerException, ForbiddenException {
+
+        // handle scope
+        boolean isInRole = false;
+        if ("system".equals(scope)) {
+            // check role
+            isInRole = context.isUserInRole(scope + "/" + role);
```

Comment:
we know scope here because of if statement, so no need to concat scope variable with slash